variables.tf

```diff
@@ -19,15 +19,18 @@ variable "automate_route_table_id" {}
 
 variable "automate_tag" { default = "terraform_automate" }
 
+# unique identifier for this instance of Chef Automate
+variable "automate_instance_id" { default = "override_me" }
+
 variable "aws_instance_type" {
   default = "m4.xlarge"
 }
 variable "aws_ami_user" {
   default = "ec2-user"
 }
 
-variable "aws_ami_rhel" { default = "ami-775e4f16" }
-
+variable "aws_ami_rhel" { default = "ami-775e4f16" } #rhel72 US-west2
```

Comment:
Recommend swapping this out for `ami-a3fa16c3`  (RHEL-7.2_HVM-20160301-x86_64-3-Hourly2-GP2)